examples/nextjs-app/lucia.d.ts

```diff
@@ -1,6 +1,6 @@
 /// <reference types="lucia-auth" />
 declare namespace Lucia {
-	type Auth = import("./auth/lucia.js").Auth;
+	type Auth = import("./auth/lucia").Auth;
```

Comment:
Does this change break the import, or does it work fine in Next.js?

Reply:
This works well in my project.

<img width="743" alt="CleanShot 2023-05-28 at 16 02 50@2x" src="https://github.com/pilcrowOnPaper/lucia/assets/103510098/16658ec3-521b-4311-9806-9e3771671cf4">

<img width="585" alt="CleanShot 2023-05-28 at 16 04 23@2x" src="https://github.com/pilcrowOnPaper/lucia/assets/103510098/1af68db1-5b80-44ca-8435-6436320c1a81">

Is there any particular reason for using the `.js` suffix?
(At first, I thought it was migrated from other frameworks)

Reply:
Don't remember, but the default TS resolution requires `.js`